src/python/pants/option/options_bootstrapper.py

```diff
@@ -160,13 +165,23 @@ def verify_configs_against_options(self, options):
     :param options: Fully bootstrapped valid options.
     :return: None.
     """
+    def is_valid_option(prefix, option, valid_options_under_scope):
+      if option in valid_options_under_scope:
+        return True
+      if prefix and '_'.join((prefix, option)) in valid_options_under_scope:
+        return True
+      return False
+
     error_log = []
+    global_scopes = (
+      # Consider inherited scopes when verifying options.
```

Comment:
s/inherited/subsumed/ ?  You're not considering all inherited scopes (which would be all scopes).

Reply:
Now that I think of it, why is this necessary? Haven't we already transformed the config so it looks as if these options are in the global section?

Reply:
yeah, this is a remnant of an earlier approach - no longer needed and yanked.